community.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Community
+description: How to contribute to the Clowder community.
+background: '/PATH_TO_IMAGE'
+---
+
+There are many ways to get involved in the Clowder community. Below we have identified a few ways and pointers to get started. *The easiest way to get more information is to join us on [Slack][slack].* Before participating, please take a momente to [read our Code of Conduct](https://github.com/clowder-framework/clowder/blob/develop/CODE_OF_CONDUCT.md).
```

Comment:
do we want to point to the master branch for conduct?

Reply:
Thanks. I really think we should change the default to `master`. This keeps happening to me. Even when I remember I still have to manually change the branch. With our current workflow I think having master would be best and easier.